api/src/test/java/org/apache/iceberg/TestHelpers.java

```diff
@@ -62,6 +70,54 @@ public static long waitUntilAfter(long timestampMillis) {
     return current;
   }
 
+  /** wait for fixed duration */
+  public static void delayInMilliseconds(String message, long delay, boolean useSameThread) {
```

Comment:
I don't think we want to use Awaitility to just wait a certain amount of time for a true condition to happen